### DISCUSSION TITLE
Version 0.6.0

### DISCUSSION THREAD

**Discussion by ahyatt** [ahyatt/ekg#150] [Announcements]
- Use note-based "memory" for LLM note additions (in `ekg-llm`).  This significantly increases the usefulness of the LLM responses.
- Add Markdown-mode inline tags.

<hr /><em>This discussion was created from the release <a href='https://github.com/ahyatt/ekg/releases/tag/0.6.0'>Version 0.6.0</a>.</em>

**Comment by oatmealm** [comment]
Does it use notes with special tags for this purpose?

**Reply by ahyatt** [reply]
For the memory?  No, it works by using previous notes with the same (non-special) tags as the note you are writing.  So no need to use special tags, although of course you can alter the instructions by using the "prompt" special tag (see the docs for instructions on how to do this).

**Reply by oatmealm** [reply]
if I understand correctly all the notes with the same set of tags will be grouped and sent together?

**Comment by ahyatt** [comment]
It doesn’t have to be the exact same set of tags, it sends notes that have
any of the tags of the note.  And just enough to cover (by default) 50% of
the model’s context window.

**Comment by oatmealm** [comment]
Works for me now with Ollama as well.
